srcpkgs/the-way/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'the-way'
+pkgname=the-way
+version=0.14.4
+revision=1
+build_style=cargo
+short_desc="Code snippets manager for your terminal"
+maintainer="jgart <jgart@dismail.de>"
+license="MIT"
+homepage="https://github.com/out-of-cheese-error/the-way"
+distfiles="https://github.com/out-of-cheese-error/${pkgname}/archive/refs/tags/v${version}.tar.gz"
```

Comment:
```suggestion
distfiles="https://github.com/out-of-cheese-error/the-way/archive/refs/tags/v${version}.tar.gz"
```

Reply:
I force pushed the changes so resolving this.